tools/setup.py

```diff
@@ -15,9 +14,11 @@ class SceSetupTool:
     color = Colors()
     devnull = open(os.devnull, 'wb')
     docker_is_running = True
+    sce_path = "" # passed in through __init__
```

Comment:
as your comment explains, `self.sce_path` will be set in `__init__`, so you don't need to define it here